RELEASE.md

```diff
@@ -0,0 +1,37 @@
+# Release Process
+
+Pack follows a 6 week release cadence, composed of 3 phases:
+  - [Development](#development)
+  - [Feature Complete](#feature-complete)
+  - [Release Finalization](#release-finalization)
+
+## Development
+Our development flow is detailed in [Development](DEVELOPMENT.md)
+
+## Feature Complete
+### Process
+5 business days prior to a scheduled release, we enter `feature complete`. A release branch (in the form `release/<VERSION>`) is created, and used for `UAT` (User Acceptance testing).
+
+During this period, relevant changes may be merged into the release branch, based on assessment by the [CCB](#ccb) of the impact, effort and risk of including the changes. Any other change may get merged into `main` through the normal process, and will make it into the next release.
+
+### Roles
+#### CCB
+The `CCB` (Change Control Board) are the `pack` [maintainers][maintainers].
```

Comment:
```suggestion
#### Change Control Board
The Change Control Board (`CCB`) are the `pack` [maintainers][maintainers].
```

Reply:
Resolved in becdf38e75f60d7a9553381dfd4b8f9bb928c21f